src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -43,6 +43,12 @@ public partial class Application
 		private ApplicationTheme? _requestedTheme;
 		private bool _themeSetExplicitly = false;
 
+		static Application()
+		{
+			ApiInformation.RegisterAssembly(typeof(Application).Assembly);
+			ApiInformation.RegisterAssembly(typeof(Windows.Storage.ApplicationData).Assembly);

```

Comment:
There were 3 assemblies before, there seem to be 2 registered now?

Reply:
The other one is registered directly ApiInformation (it's itself)